frontend/integration-tests/tests/environment.scenario.ts

```diff
@@ -89,7 +89,7 @@ describe('Interacting with the environment variable editor', () => {
       expect(resourceText).toEqual(valueFrom);
       expect(prefixText).toEqual(prefix);
     } else {
-      expect(resourceText).toEqual('httpd');
+      expect(resourceText).toEqual('container');
```

Comment:
Also, I wanted to point out that since we have changed the default container image name in the YAML from 'httpd' to 'container', the update has also been made in the `environment.scenario.ts` file.
Without this change the "Delete ConfigMap Test" is failing.